extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTask.java

```diff
@@ -207,7 +272,7 @@ public TaskStatus run(final TaskToolbox toolbox) throws Exception
 
       // Start up, set up initial offsets.
       final Object restoredMetadata = driver.startJob();
-      final Map<Integer, Long> nextOffsets = Maps.newHashMap();
+      nextOffsets.clear();
```

Comment:
Yes you're correct. At the time, when I moved nextOffsets from local variable to class field, I hadn't checked if this method would be called multiple times or not so I put the clear in here to be sure I didn't change behavior, but looking at it now I don't think it's necessary.